app/lib/backend_api_logic/service_extension.rb

```diff
@@ -17,28 +19,38 @@ module ServiceExtension
       end, class_name: 'Metric'
     end
 
-    def first_backend_api
-      @backend_api ||= find_or_create_first_backend_api!
+    class BackendApiProxy
+      attr_reader :service
+      delegate :account, :backend_apis, :backend_api_configs, to: :service
+
+      def initialize(service)
+        @service = service
+      end
+
+      def backend_api_config
+        @backend_api_config ||= backend_api.backend_api_configs.first ||
+                                backend_api_configs.build(path: '', backend_api: backend_api)
+      end
+
+      def backend_api
+        return unless account
+        @backend_api ||= backend_api_configs.first&.backend_api || account.backend_apis.build(name: service.system_name, name: "#{service.name} Backend API", description: "Backend API of #{service.name}")
```

Comment:
So under RU, the workflow is for API endpoint and UI

1. Create a Backend API
2. Create a Service
3 Link the backend API to this service

When the RU is off

1. Create the Service, the backend api is there by default

So we need to change that there too